src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:34679/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:34679/project/issues?id=sample&branch=feature2&issues=AWDAW4fIw93VfTu80wct&open=AWDAW4fIw93VfTu80wct)

[ISSUE_KEY]: AWDAW4fIw93VfTu80wct